Bio/SeqFeature.py

```diff
@@ -851,6 +851,16 @@ def __iter__(self):
             for i in range(self._start, self._end):
                 yield i
 
+    def __eq__(self, other):
+        """Check if start, end and strand of this location are the same as in the other location"""
+        if not isinstance(other, FeatureLocation):
+            return False
+        return self._start == other.start and self._end == other.end and self._strand == other.strand
```

Comment:
What data types are `.ref` and `.ref_db`? I don't see them used in any other tests.